### ISSUE TITLE
Re-enable beta support for beta models

### ISSUE THREAD

**Issue by MIchaelMainer** [microsoftgraph/msgraph-sdk-php#317]
Consider this a quick fix until we separate out beta and v1.0 models from the core Microsoft Graph client. 

- [x] Create a yaml build pipeline to generate beta PHP models with namespace support.
- [ ] Create GH action to create the GH release for beta (this will automatically create a release for packagist.org. Let's not use Azure Pipelines for this any longer which removes the need to use an access token).
- [x] Make the beta models available along with the v1.0 models.
- [x] Update the readme to reflect how we can use the beta models. 
[AB#7367](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/7367)

**Comment by MIchaelMainer** [comment]
We've added beta models into the main branch which should unblock users that need the beta models.

**Comment by SilasKenneth** [comment]
Duplicate of #517
